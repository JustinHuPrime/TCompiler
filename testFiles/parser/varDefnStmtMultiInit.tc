module foo;

void bar(int arg) {
  int i = arg * 2, j = arg * 3;
}