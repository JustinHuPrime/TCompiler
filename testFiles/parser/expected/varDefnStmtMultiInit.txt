testFiles/parser/input/varDefnStmtMultiInit.tc (code):
FILE(1, 1, STAB(ENTRY(bar, FUNCTION(testFiles/parser/input/varDefnStmtMultiInit.tc, 3, 1, void(int)))), MODULE(1, 1, ID(1, 8, foo, REFERENCES())), FUNDEFN(3, 1, KEYWORDTYPE(3, 1, void), ID(3, 6, bar, REFERENCES(testFiles/parser/input/varDefnStmtMultiInit.tc, 3, 1)), KEYWORDTYPE(3, 10, int), ID(3, 14, arg, REFERENCES()), STAB(ENTRY(arg, VARIABLE(testFiles/parser/input/varDefnStmtMultiInit.tc, 3, 10, int))), COMPOUNDSTMT(3, 19, STAB(ENTRY(i, VARIABLE(testFiles/parser/input/varDefnStmtMultiInit.tc, 4, 7, int)), ENTRY(j, VARIABLE(testFiles/parser/input/varDefnStmtMultiInit.tc, 4, 20, int))), VARDEFNSTMT(4, 3, KEYWORDTYPE(4, 3, int), ID(4, 7, i, REFERENCES(testFiles/parser/input/varDefnStmtMultiInit.tc, 4, 7)), ID(4, 20, j, REFERENCES(testFiles/parser/input/varDefnStmtMultiInit.tc, 4, 20)), BINOPEXP(4, 11, MUL, ID(4, 11, arg, REFERENCES(testFiles/parser/input/varDefnStmtMultiInit.tc, 3, 10)), LITERAL(4, 17, UBYTE(2))), BINOPEXP(4, 24, MUL, ID(4, 24, arg, REFERENCES(testFiles/parser/input/varDefnStmtMultiInit.tc, 3, 10)), LITERAL(4, 30, UBYTE(3)))))))
